cleaners/firefox.xml

```diff
@@ -18,51 +18,133 @@
+  - 5umya027.Test (Surely renamed...)
+  
+  So I guess the line should be:
+  <running type="pathname" search=glob">~/.mozilla/firefox/*/lock</running>
+  
+  But as we already check if the executable is running, I prefer to remove this line...
+  -->
+  <var name="Browser">
+    <!-- The "Browser", the forked file later should serve! -->
+    <value os="windows">Mozilla\Firefox</value>
+    <!-- If you already wanna give the universal cleaner a try: Remove the comment-marks from the following block... -->
+    <!--
+    <value os="windows">Moonchild Productions\Pale Moon</value>
+    <value os="windows">Mozilla\SeaMonkey</value>
+    <value os="windows">Waterfox</value>
+    -->
+    <!-- Linux $$Browser$$ must be figured out for each browser, too! Tobias. -->
+    <value os="linux">mozilla/firefox</value>
+    <!-- If you already wanna give the universal cleaner a try: Remove the comment-marks from the following block... -->
+    <!-- The following values are just guesses... -->
+    <!--
+    <value os="linux">seamonkey</value>
+    <value os="linux">waterfox</value>
+    -->
+  </var>
   <var name="base">
-    <value os="windows">%AppData%\Mozilla\Firefox</value>
-    <value os="linux">~/.mozilla/firefox</value>
-    <value os="freebsd">~/.mozilla/firefox</value>
-    <value os="openbsd">~/.mozilla/firefox</value>
+    <value os="windows">%AppData%\$$Browser$$</value>
+    <value os="linux">~/.$$Browser$$</value>
+    <value os="freebsd">~/.$$Browser$$</value>
+    <value os="openbsd">~/.$$Browser$$</value>
   </var>
   <var name="profile">
-    <value search="glob" os="windows">%AppData%\Mozilla\Firefox\Profiles\*</value>
-    <value search="glob" os="linux">~/.mozilla/firefox/*</value>
-    <value search="glob" os="freebsd">~/.mozilla/firefox/*</value>
-    <value search="glob" os="openbsd">~/.mozilla/firefox/*</value>
+    <value search="glob" os="windows">$$base$$\Profiles\*</value>
+    <value search="glob" os="linux">~/.$$Browser$$/*</value>
+    <value search="glob" os="freebsd">~/.$$Browser$$/*</value>
+    <value search="glob" os="openbsd">~/.$$Browser$$/*</value>
+  </var>
+  <var name="localprofile">
+    <value search="glob" os="windows">%LocalAppData%\$$Browser$$\Profiles\*</value>
+    <!-- Unknow by other OSes, yet! Tobias B. Besemer will do Linux Part, later! -->
+    <!--
+    <value search="glob" os="linux">~/.$$Browser$$/*</value>
+    <value search="glob" os="freebsd">~/.$$Browser$$/*</value>
+    <value search="glob" os="openbsd">~/.$$Browser$$/*</value>
+    -->
   </var>
   <option id="backup">
     <label>Backup files</label>
     <description>Delete the backup files</description>
+    <!-- Why not delete them all?
     <action command="delete" search="glob" path="$$profile$$/bookmarkbackups/*.json"/>
     <action command="delete" search="glob" path="$$profile$$/bookmarkbackups/*.jsonlz4"/>
+    -->
+    <action command="delete" search="walk.files" path="$$profile$$/bookmarkbackups/"/>
```

Comment:
OK

Reply:
So lines 113-116 (commented out) can be removed

Reply:
K, will remove this, but it's a backup-folder... So why just pick single files out of the backups when the open is "Remove (all) backups" ???